app/Http/Controllers/Api/ComponentController.php

```diff
@@ -124,7 +124,7 @@ public function update(Component $component)
                 Binput::get('link'),
                 Binput::get('order'),
                 Binput::get('group_id'),
-                (bool) Binput::get('enabled'),
+                Binput::get('enabled') !== null ? (bool) Binput::get('enabled') : $component->enabled,
```

Comment:
I think that this can be:

```php
(bool) Binput::get('enabled', $component->enabled)
```

Reply:
Yes indeed.. That would be better, updated in new commit/squash